src/solver/dense.jl

```diff
@@ -279,8 +279,9 @@ for (bname, fname, elty) in ((:cusolverDnSorgqr_bufferSize, :cusolverDnSorgqr, :
                                Cint, Ptr{$elty}, Ptr{$elty}, Cint, Ptr{Cint}),
                               cusolverDnhandle[1], m, n, k, A,
                               lda, tau, buffer, bufSize[], devinfo))
-            if devinfo[1] < 0
-                throw(ArgumentError("The $(devinfo[1])th parameter is wrong"))
+            info = collect(devinfo)
```

Comment:
how large is the `info` array? we could work around the `allowscalar` issue with an `unsafe_getindex` call or something, which might be better if info is large

Reply:
As I understand it is a single integer stored on the gpu that gives the success of the algorithm. Here we make a CuArray for that, but is there some way to just point to a single integer?

Reply:
Ah I see. Yeah that's fine then. We could theoretically have something like `CuRef` but I don't think it's worth it for this case.